<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelObjectEditor</class>
 <widget class="QDockWidget" name="LevelObjectEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Editor</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="group_selectLevelObjects">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Level Object</string>
    </property>
    <widget class="QListWidget" name="list_levelObjects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_levelObjectProperties">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Object Properties</string>
    </property>
    <widget class="QTabWidget" name="transformTabs">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>30</y>
       <width>200</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="Translate">
      <attribute name="title">
       <string>Translate</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>181</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="translateLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_translateY">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_translateX">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_translateY">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-99990.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
          <property name="singleStep">
           <double>2.250000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_translateX">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-99990.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
          <property name="singleStep">
           <double>2.250000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_translateZ">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Z</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_translateZ">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-99990.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
          <property name="singleStep">
           <double>2.250000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Rotate">
      <attribute name="title">
       <string>Rotate</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="translateLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_rotateY">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_rotateX">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_rotateY">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-359.000000000000000</double>
          </property>
          <property name="maximum">
           <double>359.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.025000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_rotateX">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-359.000000000000000</double>
          </property>
          <property name="maximum">
           <double>359.990000000000009</double>
          </property>
          <property name="singleStep">
           <double>0.025000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_rotateZ">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Z</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_rotateZ">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-359.000000000000000</double>
          </property>
          <property name="maximum">
           <double>359.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.025000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Scale">
      <attribute name="title">
       <string>Scale</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="translateLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_scaleY">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_scaleX">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_scaleY">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99990.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
          <property name="singleStep">
           <double>0.005000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_scaleX">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99990.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
          <property name="singleStep">
           <double>0.005000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_scaleZ">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Z</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_scaleZ">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99990.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
          <property name="singleStep">
           <double>0.005000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
